CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A system for automatically declaring a bot's guild count on the bot lists
   top.gg, bots.gg and discordbotlist.com. See [yuyo.list_status][] for more
   information.
+- `"asgi"` feature flag for ensuring this installs with the dependencies required to run
```

Comment:
Not that it's recommended you still have your own pinned tanjun version as this doesn't pin